AMP_CACHE_GUIDELINES.md

```diff
@@ -0,0 +1,85 @@
+9. Provides a faithful visual and UX reproduction of source document, although visually imperceptible changes in image resolution or compression rate are acceptable.  
+
+10. No obstruction of the contents of the original AMP document. 
+
+11. No changes to ads, access, analytics, or similar.
+
+12. Respects all resource deletions within a reasonable timeframe.
+
+13. Uses a stale-while-revalidate caching model and reasonable min cache time (such as in the order of single digit minutes) is allowed. For example, as per [Google AMP Cache updates](https://developers.google.com/amp/cache/overview#google-amp-cache-updates), "[the cache] uses the origin's caching headers, such as Max-Age, as hints in deciding whether a particular document or resource is stale. When a user makes a request for something that is stale, that request causes a new copy to be fetched, so that the next user gets fresh content.":
+
+    8. The cache is allowed to serve stale content independent of HTTP caching headers. It must make reasonable efforts to keep the cache contents fresh and must revalidate content after serving stale responses.
+
+    9. Observes the [cache-response-directive = public, max-age](http://www.freesoft.org/CIE/RFC/2068/164.htm) source caching headers.
+
+## Guidelines: Crawling 
+
+* [robots.txt](https://cdn.ampproject.org/robots.txt): Content should be served from the Cache with either the same robot rules as the origin content or stricter rules. See [https://cdn.ampproject.org/robots.txt](https://cdn.ampproject.org/robots.txt) for an example.
+
+## Guidelines: Optional Extensions
+
+* Participation in [AMP Ads For AMP Pages ("A4A")](https://github.com/ampproject/amphtml/issues/3133) advertisement signing.
+
+# References
+
+## [Google AMP Cache](https://developers.google.com/amp/cache/)
+
+* [Google AMP Cache Overview](https://developers.google.com/amp/cache/overview)
+
+* The [Google AMP Cache](https://developers.google.com/amp/cache/) is a proxy-based content delivery network for delivering all valid AMP documents.
+
+* It fetches AMP HTML pages, caches them, and improves page performance automatically. 
+
+* When using the Google AMP Cache, the document, all JS files and all images load from the same origin, which is using [HTTP 2.0](https://http2.github.io/) for maximum efficiency.
+
+* The cache also comes with a built-in [validation system](https://github.com/ampproject/amphtml/tree/master/validator) which confirms that the page works, and that it doesn’t depend on external resources. 
+
+* The validation system runs a series of assertions confirming that the page’s markup meets the AMP HTML specification.
+
+* [Google AMP Cache FAQ](https://developers.google.com/amp/cache/faq)
+
+* [Google AMP Cache updates](https://developers.google.com/amp/cache/overview#google-amp-cache-updates)
+
+* [Crawling Google AMP Cache URLs - How-to for search engines](https://docs.google.com/document/d/1V_uLHoa48IlbFl7_3KWT_1JmCf6BnFtt3S_oR4UsasQ/edit?usp=sharing)
```

Comment:
@ithinkihaveacat The immediate priority is to get this PR through to open the guidelines up for further iteration (which is highly encouraged).